### PR TITLE
[Promotion] Fix promotion display

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

Previously, the amount for channels has not been displayed

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/BuildChannelBasedPromotionActionFormSubscriber.php [hunk 1]
```diff
@@ -109,6 +109,8 @@ private function createConfigurationFieldForChannel(
             'currency' => $channel->getBaseCurrency()->getCode(),
         ];
 
+        $data = empty($data)git?:$data[$channel->getCode()];
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/BuildChannelBasedPromotionActionFormSubscriber.php]: `git?`? 😄

File: src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/BuildChannelBasedPromotionActionFormSubscriber.php [hunk 2]
```diff
@@ -109,6 +109,8 @@ private function createConfigurationFieldForChannel(
             'currency' => $channel->getBaseCurrency()->getCode(),
         ];
 
+        $data = empty($data)?:$data[$channel->getCode()];
```
Comment (Zales0123) [inline src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/BuildChannelBasedPromotionActionFormSubscriber.php]: Maybe add some spaces in this line :(

Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/BuildChannelBasedPromotionActionFormSubscriber.php]: There aren't anyspecs for this class :(

Comment (Zales0123) [inline src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/BuildChannelBasedPromotionActionFormSubscriber.php]: *Spaces* my friend, this white signs that make code more readable 😄 
```php
$data = empty($data) ?: $data[$channel->getCode()];
```

**Comment by michalmarcinkowski** [comment]
Already fixed.

**Pull request closed without merging**